bundle/regal/rules/style/prefer_snake_case_test.rego

```diff
@@ -43,9 +56,26 @@ test_success_snake_cased_multiple_some_declaration if {
 	r == set()
 }
 
+test_fail_camel_cased_function_argument if {
```

Comment:
Do we want a `_function_argument` success case for completeness?

Reply:
Sure, I'll add one before merging 👍